renovate.json

```diff
@@ -0,0 +1,53 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "github>konflux-ci/mintmaker//config/renovate/renovate.json",
+    "config:best-practices"
```

Comment:
Lets use shared config once merged https://github.com/app-sre/shared-pipelines/pull/20

Reply:
I already subscribed to the PR. :shipit: